src/components/Search/SearchRouter/SearchRouterInput.tsx

```diff
@@ -1,34 +1,30 @@
 import React, {useState} from 'react';
 import BaseTextInput from '@components/TextInput/BaseTextInput';
 import useThemeStyles from '@hooks/useThemeStyles';
-import variables from '@styles/variables';
 import CONST from '@src/CONST';
 
 type SearchRouterInputProps = {
-    isFullWidth: boolean;
-    onChange: (searchTerm: string) => void;
+    text: string;
+    setText: (searchTerm: string) => void;
+    updateSearch: (searchTerm: string) => void;
     onSubmit: () => void;
 };
 
-function SearchRouterInput({isFullWidth, onChange, onSubmit}: SearchRouterInputProps) {
+function SearchRouterInput({text, setText, updateSearch, onSubmit}: SearchRouterInputProps) {
     const styles = useThemeStyles();
 
-    const [value, setValue] = useState('');
-
     const onChangeText = (text: string) => {
-        setValue(text);
-        onChange(text);
+        setText(text);
+        updateSearch(text);
     };
 
-    const modalWidth = isFullWidth ? styles.w100 : {width: variables.popoverWidth};
-
     return (
         <BaseTextInput
-            value={value}
+            value={text}
```

Comment:
why not keep `value` to have `value=value`? its less names, less hoops to jump when reading the code?